Sources/OpenAPIRuntime/Conversion/Configuration.swift

```diff
@@ -113,9 +119,11 @@ public struct Configuration: Sendable {
     ///   - multipartBoundaryGenerator: The generator to use when creating mutlipart bodies.
     public init(
         dateTranscoder: any DateTranscoder = .iso8601,
-        multipartBoundaryGenerator: any MultipartBoundaryGenerator = .random
+        multipartBoundaryGenerator: any MultipartBoundaryGenerator = .random,
+        decodingErrorHandler: (any DecodingErrorHandler)? = nil
```

Comment:
Looks good, please just add a deprecated 2-parameter version of this initializer in Deprecated.swift, as adding a param is technically a source break. Similar to https://github.com/apple/swift-openapi-runtime/pull/102#discussion_r1547909598